.github/release-drafter-mongodb.yml

```diff
@@ -1,8 +1,8 @@
-name-template: 'MongoDB - v$RESOLVED_VERSION'
```

Comment:
You are editing the drafter-mongodb.yml, all these changes here need to be revert